tests/jballerina-unit-test/src/test/resources/test-src/functions/functions_with_included_record_parameters_negative.bal

```diff
@@ -258,3 +258,23 @@ function fn4(*Record4 record4) {
 function functionWithIncludedRecordParam18() {
     fn4(rec = {}, record4 = {}); // error
 }
+
+type Coo int;
+
+function fn(*Boo f) {
+}
+
+class MyClass {
+    function fn(*Boo f) {
+    }
+
+    function fn2(*Coo f) {
+    }
+}
+
+function testInvokingFunctionContainingIncludedRecordParamOfTypeOtherThanRecordOrUndefinedUsingNamedArg() {
+    fn(f = {});
+    MyClass obj = new;
+    _ = obj.fn(f = {});
+    _ = obj.fn2(f = {}); // No errors will be logged for these lines, but we are testing the call path
```

Comment:
Shall we add this comment before L276?

Reply:
Added in [cae7a97](https://github.com/ballerina-platform/ballerina-lang/pull/37914/commits/cae7a97605007d74888d8dd206adff3d26b4e20a)